Upgrade tinymce

rijkvanzanten opened the pull request:
Upgrade TinyMCE from 5 to 6.

Closes https://github.com/directus/directus/issues/16062
Closes #16626 (as replacement)

rijkvanzanten committed:
Upgrade tinymce

azrikahar commented:
@paescuj although #16626 might be a little while ago and you may not have the full context anymore, but would you mind taking a peek here to see whether there's any of your additional tweaks that still needs to be carried over so we can consolidate them? 😁

paescuj commented:
> @paescuj although #16626 might be a little while ago and you may not have the full context anymore, but would you mind taking a peek here to see whether there's any of your additional tweaks that still needs to be carried over so we can consolidate them? 😁

Yup, this is what I've inteded to do 😁👍

paescuj committed:
Omit 'skin_url' instead of setting it to false

The option expect a string value. Instead of setting it to 'false' we
can just completly omit it.
Otherwise we get the following warning:
> Invalid value passed for the skin_url option. The value must be a string.

Although not explicitly mentioned in the migration doc, the stricter check / warning has
probably been introduced with v6.

paescuj committed:
Disable pasting of images

From
https://www.tiny.cloud/docs/tinymce/6/migration-from-5x/#plugins-paste:

> A paste default has changed. The paste_data_images option now defaults to true.
> When paste was a plugin, this option was, by default, set to false, which prevented images being pasted from the local machine

rijkvanzanten committed:
Merge branch 'main' into feat/tinymce

rijkvanzanten committed:
Update toolbar option names

paescuj reviewed (approved):
![the-muppets-kermit-the-frog](https://user-images.githubusercontent.com/5363448/230979570-970d87a9-3bce-4fb9-aae4-e5ef59e48d76.gif)
